dev/sparktestsupport/modules.py

```diff
@@ -72,7 +78,10 @@ def __init__(self, name, dependencies, source_file_regexes, build_profile_flags=
         self.dependent_modules = set()
         for dep in dependencies:
             dep.dependent_modules.add(self)
-        all_modules.append(self)
+        if name == "hive-thriftserver" and hadoop_version == "hadoop3.2":
```

Comment:
Used to skip hive-thriftserver module for hadoop-3.2.  Will revert this change once we can merge.

Reply:
Maybe leave a TODO here just to try to make sure that doesn't get lost

Reply:
Add to release note?